lib/DebugProcess.php

```diff
@@ -14,6 +14,11 @@ public function __construct(Logger $logger, Bootstrapper $bootstrapper = null) {
     }
 
     protected function doStart(Console $console): \Generator {
+        if ($console->isArgDefined("restart")) {
+            $this->logger->critical("You cannot restart a debug aerys instance via command");
```

Comment:
Because we don't have separate processes for debug instances. Restarting is actually killing workers and starting them again.